Content.Server/Fax/FaxSystem.cs

```diff
@@ -399,6 +445,15 @@ public void Copy(EntityUid uid, FaxMachineComponent? component = null)
         if (sendEntity == null)
             return;
 
+        if (HasComp<MobStateComponent>(sendEntity))
+        {
+            var damageSpec = new DamageSpecifier(_proto.Index<DamageGroupPrototype>("Brute"), 300);
+            _damageable.TryChangeDamage(sendEntity, damageSpec);
+            _popupSystem.PopupEntity(Loc.GetString("fax-machine-popup-error", ("target", uid)), uid, PopupType.LargeCaution);
+            return;
+        }
+
+
```

Comment:
This should be raising an event and handled elsewhere, also Brute type needs validating rather than using a magic string here.

Reply:
moved this and gave it an event, brute and damage amount now set in component.